src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.jsx

```diff
@@ -244,6 +244,9 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
       expect(document.activeElement).to.have.tagName('h1');
     });
 
+    expect(await screen.findByText(/This appointment occurred in the past./i))
```

Comment:
Can we also check that this doesn't exist in a test that uses a future appointment?